.github/workflows/multiarch.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Multi-arch builds
+
+on:
+  pull_request:
+
+jobs:
+  apply-suggestions-commits:
+    name: Check the multi-arch builds
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - name: Set up QEMU
```

Comment:
Can we add documentation why QEMU is needed?

Reply:
I’ve added a brief mention in the step name, and in the commit message (and PR description).